config/200-clusterrole.yaml

```diff
@@ -17,6 +17,9 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: tekton-triggers-admin
 rules:
+  - apiGroups: [""]
+    resources: ["serviceaccounts"]
+    verbs: ["get", "list", "watch"]
```

Comment:
per the conference call this may be the wrong spot ... will need to look some more, unless a reviewer has a pointer before I find it.

Reply:
All of our examples event listeners use the example service account here: https://github.com/tektoncd/triggers/tree/master/examples/role-resources

Reply:
ah thanks @dibyom !

Reply:
the undoing of this change and the updates to https://github.com/tektoncd/triggers/tree/master/examples/role-resources to add read access to serviceaccounts pushed